src/autoComplete/index.tsx

```diff
@@ -219,11 +231,15 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
           defaultValue={String(this.state.query)}
           addon={this.props.inputAddon}
           inputRef={this.inputRef}
-          autoCorrect="off"
-          autoComplete="off"
+          autoCorrect={this.props.autoCorrect}
+          autoComplete={this.props.autoComplete}
```

Comment:
autoComplete should always be off IMO

Reply:
On the Autocomplete, I agree. Is that a problem @dagatsoin ?

Reply:
No.